### PR TITLE
Implement basic login

### PR DESCRIPTION
**Pull request by PhanVanDung2407**
image

### CODE REVIEW

**Comment by PhanVanDung2407** [comment]
ready

**Comment by PhanVanDung2407** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+    if user && user.authenticate(params[:session][:password])
+      log_in user
+      redirect_to user
+    else
+      flash.now[:danger] = I18n.t(".users.new.invalid_email_password")
+      render "new"
```
Comment (dothidiemthao) [inline app/controllers/sessions_controller.rb]: dùng render :new nha

File: app/helpers/sessions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
```
Comment (dothidiemthao) [inline app/helpers/sessions_helper.rb]: dùng "current_user.present?" thay cho "!current_user.nil?"

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<% provide(:title, "Log in") %>
```
Comment (dothidiemthao) [inline app/views/sessions/new.html.erb]: i18n này

**Comment by dothidiemthao** [comment]
commented

**Comment by PhanVanDung2407** [comment]
ready

**Review by dothidiemthao — APPROVED** [review]

**Comment by dothidiemthao** [comment]
merged

**Pull request merged**